RTProtocol.h

```diff
@@ -381,39 +380,39 @@ class DLL_EXPORT CRTProtocol
     char* GetErrorString();
 
 private:
-    bool SendString(const char* pCmdStr, int nType);
-    bool SendCommand(const char* pCmdStr);
+    bool SendString(const char* cmdStr, int type);
+    bool SendCommand(const char* cmdStr);
     bool SendCommand(const std::string& cmdStr, std::string& commandResponseStr, unsigned int timeout = cWaitForDataTimeout);
-    bool SendXML(const char* pCmdStr);
+    bool SendXML(const char* cmdStr);
     const char * ReadSettings(const std::string& settingsType);
     bool ReceiveCalibrationSettings(int timeout = cWaitForDataTimeout);
 
 private:
```

Comment:
Why did we choose to keep the `m `prefix for members? Imo I think we should just remove the member variable identifier entirely, or use trailing underscore.

Reply:
This applies to all files, not just here.

Reply:
Johannes and I concluded that it makes sense to distinguish member variables vs. local variables / parameters. Just to clarify, you're suggesting to change from "mMemberVar" to "**m_MemberVar**"?

Reply:
I would actually want to advocate for `member_variable_`, but if we want to keep more to the same naming standard as we do now, then `memberVariable_` is fine.